apluslms_roman/cli.py

```diff
@@ -210,6 +212,11 @@ def add_cli_actions(parser):
         help=_("build the course (default action)"))
     build.add_argument('course', nargs='?',
         help=_("location of the course definition (default: current working dir)"))
+    build.add_argument('-s', '--steps', nargs='+',
+        help=_("select which steps to build and in which order (use either index or step name)"))
+    build.add_argument('-ls', '--list-steps',
```

Comment:
Double char short options are always problematic. What would `-ls foo` mean, is it `-l --steps foo` or `--list-steps foo`. Short options can typically be grouped when given to commands and this is behavior what could be expected by the user. I don't personally see need for a short option, as the `-s ?` is quite intuitive too (I have seen that some times, best I could find now is `kvm -device ?`, which includes bad example of an argument itself).

Reply:
Well in zsh you need to quote the ? which makes it rather hard imo. I'll make "-l" the short option.

Reply:
Yep, `?` is converted to a single character, if there exists a single character file. Shells typically pass that as is if it won't expand. E.g.

```
% echo ?; touch a; echo ?
?
a
```

Reply:
should "roman -s ?" work or is it ok if only `roman -l` and `roman build -s ?` work?

Reply:
Only build related command is`roman`. If user needs to add any options, then he is forced to write `roman build <something>`. Of course `roman -f foobar.yml` will work too, but yeah, rest is via the subcommand.